ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```diff
@@ -49,13 +52,24 @@ EOF
 ADDITIONAL_ARG=""
 # default to the first cluster in the array, unless 4.14
 if [[ "$T5_JOB_DESC" == "periodic-cnftests" ]]; then
-  ADDITIONAL_ARG="--cluster-name ${PREPARED_CLUSTER[0]} --force" 
-  if [[ "$T5CI_VERSION" == "4.14" ]]; then 
-    ADDITIONAL_ARG="--cluster-name ${PREPARED_CLUSTER[1]} --force"
-  fi 
+    ADDITIONAL_ARG="--cluster-name ${PREPARED_CLUSTER[0]} --force"
+    if [[ "$T5CI_VERSION" == "4.14" ]]; then
+        ADDITIONAL_ARG="--cluster-name ${PREPARED_CLUSTER[1]} --force"
+    fi
 else
   ADDITIONAL_ARG="-e $CL_SEARCH --exclude ${PREPARED_CLUSTER[0]} --exclude ${PREPARED_CLUSTER[1]}"
```

Comment:
Modify indent to contain 4 spaces for congruence.

Reply:
it IS 4 spaces now

Reply:
done